server_config.go

```diff
@@ -105,11 +109,22 @@ func (cfg *ServerConfig) makeCommand() func() *exec.Cmd {
 		return cfg.CommandProducer(cfg)
 	}
 
-	var cmd = strings.Split(cfg.Command, " ")
+	var cmdArgs []string
+	cmdArgs = append(cmdArgs, strings.Split(cfg.Command, " ")...)
+
 	return func() *exec.Cmd {
-		cmd := exec.Command(cmd[0], cmd[1:]...)
+		cmd := exec.Command(cmdArgs[0], cmdArgs[1:]...)
 		osutil.IsolateProcess(cmd)
 
+		// if user is not empty, and OS is linux or macos
+		// execute php worker from that particular user
+		if cfg.User != "" {
+			err := osutil.ExecuteFromUser(cmd, cfg.User)
```

Comment:
win fallback?

Reply:
ahh, crap :) You are right. On windows, this will fail to build.

Reply:
Done